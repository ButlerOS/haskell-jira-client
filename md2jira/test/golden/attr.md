# epic

## story {#P-4 score=42 updated=2024-09-07, prio=H}

- [ ] task {.end}
note
