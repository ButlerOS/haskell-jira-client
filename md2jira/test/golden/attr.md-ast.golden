Document
    { config = EmptyConfig
    , intro = []
    , epics =
        [ Epic
            { mJira = Nothing
            , title = "epic"
            , description = []
            , stories =
                [ Story
                    { mJira = Just "P-4"
                    , title = "story"
                    , description =
                        [ BulletList
                            [
                                [ Plain
                                    [ Str "‚òê"
                                    , Space
                                    , Str "task"
                                    , Span
                                        ( ""
                                        , [ "end" ]
                                        , []
                                        ) [ Space ]
                                    , SoftBreak
                                    , Str "note"
                                    ]
                                ]
                            ]
                        ]
                    , mScore = Just 42.0
                    , updated = Just 1725667200
                    }
                ]
            }
        ]
    }